vortex-java-sdk/vortex-java-sdk-iam/src/main/java/com/consoleconnect/vortex/iam/dto/UpdateConnectionDto.java

```diff
@@ -2,16 +2,17 @@
 
 import com.consoleconnect.vortex.iam.enums.ConnectionStrategryEnum;
 import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
 import lombok.Data;
 
 @Data
 public class UpdateConnectionDto {
 
   @NotBlank private String id;
 
-  private ConnectionStrategryEnum strategy = ConnectionStrategryEnum.SAML;
+  @NotNull private ConnectionStrategryEnum strategy;
 
-  private OidcConnection odic;
+  private OidcConnectionDto odic;
```

Comment:
```suggestion
  private OidcConnectionDto oidc;
```

Probably warrants more replacements but I can see this getting very confusing elsewhere.

Reply:
This parameter represents the open-id connection that receives parameters passed in by the front end and will be used in specific connections.

Reply:
The variable is still mispelled.

Reply:
Changed it to `openID`.